js/history.js

```diff
@@ -3,17 +3,80 @@ Fliplet.Registry.set('history-stack', (function() {
   var stack = [];
   var currentIndex = 0;
 
+  const columnsInfo = {
+    original: () => stack[0].data[0],
+    reference: [],
+    getReference: () => {
+      if (!columnsInfo.reference.length) {
+        columnsInfo.reference = columnsInfo.original();
+      }
+      return columnsInfo.reference;
```

Comment:
See other comment re: linter setup

```suggestion

      return columnsInfo.reference;
```